install_config/install/advanced_install.adoc

```diff
@@ -288,8 +288,8 @@ master configuration].
 |This variable specifies the size of the per host subnet allocated for pod IPs
 by
 xref:../../architecture/networking/sdn.adoc#sdn-design-on-masters[{product-title}
-SDN]. Defaults to `9` which means that a subnet of size /23 is allocated to each
-host; for example, given the default 10.128.0.0/14 cluster network, this will
+SDN]. Reasonable value is `9` which means that a subnet of size /23 is allocated to each
```

Comment:
Why reasonable? Do we mean 'suggested'?

Reply:
If it was suggested, we could just as well use the initial defaults. Since the defaults are no longer there, OpenShift team likely thinks that those values should no longer be suggested to people as something they should use without modification.